packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -8244,4 +8244,45 @@ describeWithDOM('mount', () => {
       expect(spy).to.have.property('callCount', 1);
     });
   });
+
+  describe('hooks', () => {
+    it('will run', () => {
+      const Foo = () => {
+        const [fooValue, setFooValue] = React.useState("hi");
```

Comment:
please import `useState` from `react-compat`, instead of accessing it directly off of `React`